packages/mermaid/src/docs/index.html

```diff
@@ -15,6 +15,11 @@
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     />
+    <meta property="og:title" content="Mermaid" />
+    <meta property="og:url" content="https://mermaid-js.github.io/mermaid/" />
+    <meta property="og:image" content="https://i.ibb.co/883k0pH/mermaid-logo.jpg" />
```

Comment:
```suggestion
    <meta property="og:image" content="https://mermaid-js.github.io/mermaid/img/header.png" />
```

My gut feeling is that we should instead be using https://mermaid-js.github.io/mermaid/img/header.png as an image. What do you think?

Reply:
looks good, the picture better explains the use of the mermaid. great

Reply:
> My gut feeling is that we should instead be using https://mermaid-js.github.io/mermaid/img/header.png as an image. What do you think?

done